app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -259,7 +261,32 @@ export default {
       this.conversationPanel.removeEventListener('scroll', this.handleScroll);
     },
     scrollToBottom() {
-      this.conversationPanel.scrollTop = this.conversationPanel.scrollHeight;
+      function totalMessageHeight(total, element) {
```

Comment:
Since there is lot of cases, Can we move it to a helper file ? Which has a function accepting `unreadMessages, readMessages DOM arrays` and returns the scrollTop ?
Also would be great to see some inline comments about the steps